week-03/assessment/solar-farm-starter/src/main/java/learn/solarfarm/data/SolarPanelFileRepository.java

```diff
@@ -48,8 +48,32 @@ public SolarPanel create(SolarPanel solarPanel) throws DataAccessException {
     }
 
     // TODO: add an update method (must match with interface)
+    public boolean updateFile(SolarPanel solarPanel) throws DataAccessException{
```

Comment:
Just `update()` would be a good method name. the word `File` leaks too much implementation detail.